pkg/datapath/loader/loader_test.go

```diff
@@ -78,8 +76,6 @@ func (s *LoaderTestSuite) SetUpSuite(c *C) {
 		return os.RemoveAll(tmpDir)
 	}
 
-	ctmap.InitMapInfo(option.CTMapEntriesGlobalTCPDefault, option.CTMapEntriesGlobalAnyDefault, true, true, true)
```

Comment:
Why were we calling this function here? Were calls to `InitMapInfo()` in some of the test suites unnecessary in the first place or do the changes in this PR enable us to delete them?

Reply:
These calls are removed thanks to the changes in this PR. For instance, the [`TestWrap` ](https://github.com/cilium/cilium/blob/main/pkg/datapath/loader/template_test.go#L16-L44) test calls [`WriteBPFMacros`](https://github.com/cilium/cilium/blob/main/pkg/maps/ctmap/ctmap.go#L807-L819), which in turn create the [map objects](https://github.com/cilium/cilium/blob/main/pkg/maps/ctmap/ctmap.go#L333-L346), historically requiring the global structure to be initialized.

Reply:
Ah, thanks for the context :pray:.